src/AirshipBindings.Android.Core/AirshipBindings.Android.Core.csproj

```diff
@@ -10,11 +10,12 @@
     <RootNamespace>UrbanAirship</RootNamespace>
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
-    <AssemblyName>AirshipBindings.Core</AssemblyName>
+    <AssemblyName>AirshipBindings.Android.Core</AssemblyName>
     <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
     <AndroidTlsProvider></AndroidTlsProvider>
+    <AndroidClassParser>class-parse</AndroidClassParser>
```

Comment:
This change solved a lot of my issues. The old `jar2xml` parser was having trouble with AndroidX.

Reply:
what does the change entail?

Reply:
It's a setting in the project UI. `class-parse` seems to do a better job.
![image](https://user-images.githubusercontent.com/2123928/76237207-1b36b380-61eb-11ea-8833-d1585dd028ef.png)